tests/usage.test.ts

```diff
@@ -55,6 +57,13 @@ const schemas = [
   //   schema: prototypeApplicationSchema,
   //   examples: getJSONExamples<PrototypeApplication>('prototypeApplication'),
   // },
+  // {
+  //   name: 'PostSubmissionApplication',
+  //   schema: postSubmissionApplicationSchema,
+  //   examples: getJSONExamples<PostSubmissionApplication>(
+  //     'postSubmissionApplication',
+  //   ),
+  // },
```

Comment:
:bookmark: promise to come back to this soon & make test setup more flexible :slightly_smiling_face: